packages/app/src/components/Randomizer/Randomizer.test.js

```diff
@@ -43,45 +44,45 @@ test('restaurant url is not rendered initially (no restaurant is shown)', async
 test('restaurant url is rendered after a restaurant is drawn', async () => {
   const { queryByTestId, getByTestId } = await actRender(<Randomizer />)
 
-  fireEvent.click(queryByTestId('randomizer-randomizeButton'))
-  await waitForDomChange()
-
-  const url = getByTestId('randomizer-restaurantUrl')
-  expect(url).toBeInTheDocument()
+  await act(async () => {
+    fireEvent.click(queryByTestId('randomizer-randomizeButton'))
+    await waitForDomChange()
+    const url = getByTestId('randomizer-restaurantUrl')
+    expect(url).toBeInTheDocument()
```

Comment:
Use of `await waitForDomChange()` is generally considered a code smell.
```suggestion
    await wait(() => expect(getByTestId('randomizer-restaurantUrl')).toBeInTheDocument())
```